pkg/configurator/client.go

```diff
@@ -16,53 +16,6 @@ import (
 	"github.com/openservicemesh/osm/pkg/kubernetes/events"
 )
 
-const (
```

Comment:
Since you are not deleting `client.go` and they are on the same folder, I'd suggest you change the definitions on a different commit (you could even move them to a separate file in same folder, I'm assuming `crd_client.go` is in the same folder)

Reply:
I think we need to keep the legacy client for a while. The only use for `client.go` is during the installation, we attempt to auto convert from configmap to CRD.

Until we completely remove `client.go`, we can either keep `client.go` and `crd_client.go` (I prefer this), or:

* `client` -> `configmap_client`
* `crd_client` -> `client`

Reply:
Will work on the refactor in a separate PR.